lib/dav/dav.ts

```diff
@@ -143,7 +143,7 @@ export const getFavoriteNodes = async (davClient: WebDAVClient, path = '/', davR
 export const davResultToNode = function(node: FileStat, filesRoot = davRootPath, remoteURL = davRemoteURL): Node {
 	const props = node.props as ResponseProps
```

Comment:
Maybe add `owner-id` to `ResponseProps` as `string | undefined`? Then you do not need to cast later and it is more clear that it exists